cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -283,6 +306,40 @@ func del(args *skel.CmdArgs, cniTypes typeswrapper.CNITYPES, grpcClient grpcwrap
 		return errors.Wrap(err, "del cmd: failed to load k8s config from args")
 	}
 
+	if args.Netns == "" {
```

Comment:
Is there a specific reason this check is added?

Reply:
Added comments